stories/alert/Alert.stories.tsx

```diff
@@ -0,0 +1,74 @@
+import type { Meta, StoryObj } from "@storybook/react";
+
+import React from "react";
+import { View } from "react-native";
+import { Alert } from "../../src/components";
+
+// More on how to set up stories at: https://storybook.js.org/docs/react/writing-stories/introduction#default-export
+const meta = {
+  title: "IO-App Design System/Components/Alert/Alert",
+  component: Alert,
+  parameters: {
+    // Optional parameter to center the component in the Canvas. More info: https://storybook.js.org/docs/react/configure/story-layout
+    layout: "centered"
+  },
+  argTypes: {
+    title: {
+      control: { type: "text" }
+    },
+    content: {
+      control: { type: "text" }
+    },
+    variant: {
+      control: { type: "select" },
+      options: ["success", "error", "warning", "info"]
+    }
+  },
+  args: {
+    title: "Alert Title",
+    content: "Alert content"
+  },
+  render: args => {
+    const { title, content, variant } = args;
+    const AlertWithRef = () => {
+      const viewRef = React.createRef<View>();
+      return (
+        <Alert
+          content={content}
+          title={title}
+          viewRef={viewRef}
+          variant={variant}
+        />
+      );
+    };
+    return AlertWithRef();
```

Comment:
```suggestion
    const viewRef = React.createRef<View>();
    return (
      <Alert
        content={content}
        title={title}
        viewRef={viewRef}
        variant={variant}
      />
    );
```

I think we can omit to define and call a new function letting render function return directly the component, what do you think?

Reply:
I agree, in this case it can be simplified.  However this setup is needed if instead of React.createRef we use React hooks like useRef.

Reply:
yes I know that it is needed for the usage of ref, my suggestion is to replace the function definition + directly calling the function by just returning the component in the render argument of the story's meta.

Just like it has been done [here](https://github.com/pagopa/io-app-design-system/blob/main/stories/numberpad/NumberPad.stories.tsx#L16)